app/src/main/resources/common/README.md.mustache

```diff
@@ -91,14 +91,28 @@ On a successful deployment via the following methods, the server will be availab
 {{#configServer}}* `https:/localhost:8888/casconfigserver`{{/configServer}}
 {{#discoveryServer}}* `https://localhost:8761`{{/discoveryServer}}
   
-## Executable WAR
+## Build and run CAS with an embedded server
 
-Run the server web application as an executable WAR. Note that running an executable WAR requires CAS to use an embedded container such as Apache Tomcat, Jetty, etc.
+Run the server web application as an executable WAR. 
```

Comment:
This is misleading, I think. The title for this section says: "build and run CAS with an embedded server" and you're describing executable WARs. These have nothing to do with each other. If tou want to describe an executable WAR deployment, you should start a separate section, and not modify what exists already.